__tests__/feature/editing/editingLiteral.test.js

```diff
@@ -114,6 +114,11 @@ describe('editing a literal property', () => {
     fireEvent.click(screen.getByText('ọ'))
     expect(input).toHaveValue('Foọ')
 
+    // press backspace while the focus is on the diacritic panel and make sure we are still on the edit page
+    const diacriticsPanel = document.querySelector('#diacritics-selection')
+    fireEvent.keyDown(diacriticsPanel, { key: 'Backspace', code: 8, charCode: 8 })
```

Comment:
The **code** parameter is incorrect, it should be `keyCode` with call looking like `fireEvent.keyDown(diacriticsPanel, { key: 'Backspace', keyCode: 8, charCode: 8 })`. But, here might be a case where jsdom is not replicating the browser functionality because even when I tested sending the correct backspace code (confirmed by doing a `console.log` in the `keyPressHandle` function) the Diacritics panel is still showing present and not closed.

Reply:
i noted that right below there is keyPress event that uses the same syntax: https://github.com/LD4P/sinopia_editor/blob/2103-fix-backspace-bug/__tests__/feature/editing/editingLiteral.test.js#L48

Regardless, the test passes with either syntax